Tools/jsdoc/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
       "noEmit": true,
-      "types": []
+      "types": [],
+      "preserveConstEnums": true
```

Comment:
@ggetz  I just realized, shouldn't this be false? And if we need to process manually anyway, does it matter?  Should it just be deleted?  Feel free to just put a follow-up commit in main.

Reply:
@mramato The documentation made it sounds like I should enabled the flag, but its not explicit.

> [preserveConstEnums](https://www.typescriptlang.org/tsconfig#preserveConstEnums) emits the same JavaScript for const enums as plain enums. You can then safely strip the const modifier from .d.ts files [in a build step](https://github.com/microsoft/TypeScript/blob/1a981d1df1810c868a66b3828497f049a944951c/Gulpfile.js#L144).

And the default value is different depending on whether `isolatedModules` is enabled or not.

> Default:
true if [isolatedModules](https://www.typescriptlang.org/tsconfig#isolatedModules),
false otherwise.

As long as we strip the const enums,`build-ts` passes either way, or if the flag is omitted entirely. So I assume we should omit it?

Reply:
I honestly don't know and it probably doesn't matter, but if setting this to true vs false ends up producing the same exact output for us, then it's probably better to remove it to avoid confusion in the future

Reply:
Removed in https://github.com/CesiumGS/cesium/pull/10076